libsrc/sim6502/paravirt.s

```diff
@@ -7,11 +7,14 @@
 ; int __fastcall__ write (int fd, const void* buf, unsigned count);
 ;
 
-        .export         args, exit, _open, _close, _read, _write
+        .importzp       sp
+        .export         args, _open, _close, _read, _write
 
 args            := $FFF0
-exit            := $FFF1
+;exit           := $FFF1 ; crt0.s (not affected by assert below)
```

Comment:
I think that the comment should be a little different: "; in crt0.s, doesn't need .assert below".

Reply:
Ok. Changed.